src/main/java/io/lenses/sql/udf/count_data.java

```diff
@@ -0,0 +1,16 @@
+package io.lenses.sql.udf;
+
+public class count_data implements UserDefinedAggregateFunction {
```

Comment:
This is identical to the one built in so doesn't really show much.
I'd do one called double_count - that's what I show in my demos, and do count += 2;

Reply:
sure!